src/modules/debug/dotnet.ts

```diff
@@ -0,0 +1,7 @@
+import { Runtime } from '../../clients/runtime'
+import { getIOContext } from '../utils'
+
+export default async (debugInst: string) => {
+  const runtimeClient = new Runtime(getIOContext())
+  await runtimeClient.debugDotnetApp(debugInst)
```

Comment:
How about this instance function receives a manifest as argument, then the manifest is read before this line and we can warn the user that this command can't be used if dotnet builder is not used:
```ts
  const manifest = await ManifestEditor.getManifestEditor()
  const { builders } = manifest
  if(!builders?.dotnet) {
    logger.error('This command can only be used for dotnet apps')
    process.exit()
  }

  await runtimeClient.debugDotnetApp(debugInst, manifest)
```

Reply:
I already do this check at `runtimeClient.debugDotnetApp()`, but without printing any kind of error message.  Do you want me to move it to the module instead?

Reply:
I think it would be better - the client seems a little odd to me to log errors to the stdout - the suggestion to move the manifest instantiation would prevent doing the same work two times